### PR TITLE
S04 write multiple streams in parallel

### PR DESCRIPTION
**Pull request by jkonecki**
Added example of writing to multiple streams in parallel for performance testing.

### CODE REVIEW

**Comment by yevhen** [comment]
Good one. I remember we were having a conversation around this example. Can't remember what was the result :smile:

File: Source/Example/Scenarios/S04_Write_to_stream.cs [hunk 1]
```diff
@@ -58,6 +60,40 @@ void WriteSequentiallyToExistingStream()
...
+                    var existent = Stream.TryOpen(partition);
+
+                    var stream = existent.Found
+                        ? existent.Stream
+                        : new Stream(partition);
+
+                    Console.WriteLine("Writing to new stream in partition '{0}'", partition);
+                    var stopwatch = Stopwatch.StartNew();
+
+                    for(int i=0; i<30; ++i)
```
Comment (yevhen) [inline Source/Example/Scenarios/S04_Write_to_stream.cs:81]: Can I ask you for a minor change here? Make spaces in `for` statement consistent with the rest of code.  I need to push my ReSharper settings file along with solution! :smile:

Comment (jkonecki) [inline Source/Example/Scenarios/S04_Write_to_stream.cs:81]: Sure. Will do tomorrow morning.

Comment (yevhen) [inline Source/Example/Scenarios/S04_Write_to_stream.cs:81]: Good night! :smile:

Comment (yevhen) [inline Source/Example/Scenarios/S04_Write_to_stream.cs:81]: Let's do it other way around. That's not so important, I'll fix formatting myself

**Pull request merged**